packages/radio/src/react/button.js

```diff
@@ -99,16 +68,26 @@ class Button extends React.Component {
         onClick={props._disabled ? null : this.handleClick}
         {...styles.button(allProps)}
       >
-        <div
-          role="radio"
-          aria-checked={allProps.checked}
-          tabIndex="-1"
-          onFocus={props._disabled ? null : _ => props._onFocus(props.value)}
-          ref={el => (this.circle = el)}
-          {...styles.circle(allProps)}
+        <Halo
+          appearance={
+            !allProps._isFocused && allProps._error
+              ? Halo.appearances.error
+              : Halo.appearances.default
+          }
+          shape={Halo.shapes.pill}
```

Comment:
suprised this worked - i thought you would have needed a new shape but i guess it makes sense